torch/csrc/jit/script/compiler.cpp

```diff
@@ -791,7 +808,7 @@ struct to_ir {
       // Record the type for the schema and set the Type on the Value*
       arguments.push_back(schema.arguments.at(arg_annotation_idx++));
       new_input->setType(arguments.back().type);
-      ensureLegalType((*it).ident().range(), arguments.back().type);
+      // ensureLegalType((*it).ident().range(), arguments.back().type);
```

Comment:
What's the deal with this one and the one call below?

Reply:
I forgot to delete it, thanks!